client/servers.cpp

```diff
@@ -873,27 +667,14 @@ void server_scan_finish(struct server_scan *scan)
     fc_thread_wait(&scan->meta.thr);
     fc_destroy_mutex(&scan->meta.mutex);
 
-    /* This mainly duplicates code from below "else" block.
-     * That's intentional, since they will be completely different in future
-     * versions. We are better prepared for that by having them separately
-     * already. */
-    if (scan->sock >= 0) {
-      fc_closesocket(scan->sock);
-      scan->sock = -1;
-    }
-
     if (scan->srvrs.servers) {
       fc_allocate_mutex(&scan->srvrs.mutex);
       delete_server_list(scan->srvrs.servers);
       scan->srvrs.servers = NULL;
       fc_release_mutex(&scan->srvrs.mutex);
     }
   } else {
-    if (scan->sock >= 0) {
-      fc_closesocket(scan->sock);
-      scan->sock = -1;
-    }
-
+    fcUdpScan::i()->drop();
```

Comment:
Its here ( if  not global scan)

Reply:
Oh I see, it's only used for local scans